lib/ffmpeg/movie.rb

```diff
@@ -145,6 +145,14 @@ def unsupported_streams(std_error)
       end
     end
 
+    def audio_frames
+      metadata[:frames].select { |f| f[:media_type] == "audio" }
```

Comment:
might be a bit confusing to filter only media type in case you have multiple audio streams: maybe better to return only frames and keep ffmpeg contract and do the filtering/processing in trogdor.
_(adding a layer of intermediate transformation here for this info seems to not bring a lot of value)_